src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: $color-background;
+  border: 1px solid $color-border;
+  border-radius: 5px;
+  padding: 0 16px 16px;
+  margin-right: 48px;
+  margin-bottom: 46px;
```

Comment:
follow to bem metodology
block shouldn't have margins
for margins better to make elements for container block